src/08-accessing.js

```diff
@@ -148,7 +166,9 @@ function getCityAndState(person) {}
  *  getFullAddress(person);
  *  //> "697 Pine Drive 2A, Staten Island, NY, 10306"
  */
-function getFullAddress(person) {}
+function getFullAddress(person) {
+  return `${person.address.street} ${person.address.unit}, ${person.address.city}, ${person.address.state}, ${person.address.zip}`;
```

Comment:
When using backticks we can break strings across multiple lines.  With a long string like this we should spread this out over multiple lines.